week1/ro-el/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Self Introduction</title>
+    <style>
+      table,
+      th,
+      td {
+        border: 1px solid black;
+        border-collapse: collapse;
+      }
+      td {
+        width: 100px;
+        height: 25px;
+        text-align: center;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>안녕하세요, 정경은입니다!😊</h1>
+    <h2>기본 정보</h2> 
+    Birth: 2000.04.29<br />
+    Major: 컴퓨터공학부 19학번
+    <h3>TMI</h3>
+    MBTI: ISTJ
```

Comment:
HTML에서 태그 사용 안하고 그냥 문자나 숫자를 입력해도 화면에 나타나기는 하지만, 태그를 사용하여 정보의 종류나 중요도를 표시하는 것이 좋습니다. 
```suggestion
    <h2>기본 정보</h2> 
    <ul>
      <li>Birth: 2000.04.29</li>
      <li>Major: 컴퓨터공학부 19학번</li>
    </ul>
    <h3>TMI</h3>
    <p>MBTI: ISTJ</p>
```
이렇게 하면 스크린 리더와 같은 접근성 도구를 사용하는 사용자들에게도 정보가 잘 전달이 되고, 검색 엔진 최적화에도 도움이 됩니다.

Reply:
오!! 일반 텍스트 같은 경우에 왜 태그로 감싸나 했는데 이런 이유가 있군여
감사합니당 😍